GameDev/week3/homework/solution/solutions.md

```diff
@@ -0,0 +1,93 @@
+# Assignment 1: Car Boost
+
+## Solution 1:
+
+
```

Comment:
You can format your code like this with syntax highlighting:
```cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
```
to do this, place ` ```cs` on a line, your code below that, and finally ` ``` ` to close off the code.